deployment/modules/gcp/conformance/main.tf

```diff
@@ -49,6 +54,11 @@ resource "google_project_service" "compute_engine" {
   disable_on_destroy = false
 }
 
+resource "google_project_service" "spanner_api" {
```

Comment:
Does the spanner resource proper also need a `depends_on` link to this like you added for the `google_compute_default_service_account`?

Reply:
PTAL, I moved it to a different file and added the depends_on.

Reply:
Cool, yeah, that's a better place.